index.js

```diff
@@ -1,9 +1,13 @@
 import configureEnv from "./config/configureEnv.js";
 import * as dotenv from "dotenv";
 dotenv.config();
-//test new branch
-// console.log(process.env);
 
-configureEnv().then(() => {
-  console.log(process.env);
-});
+const wrapperOnEnv = async () => {
+  try {
+    await configureEnv();
+    await console.log(process.env);
+  } catch (err) {
+    console.log(err);
+  }
+};
+wrapperOnEnv();
```

Comment:
this is fine, but we don't really need to name this and call it:

```js
(async () => {
  console.log(process.env);
  try {
});
    await configureEnv();
    await console.log(process.env);
  } catch (err) {
    console.log(err);
  }
})();
```

Reply:
got it